Resources/Prototypes/_Moffstation/GameRules/listening_outpost.yml

```diff
@@ -18,6 +18,7 @@
   components:
   - type: GameRule
     minPlayers: 25
+    cancelPresetOnTooFewPlayers: true
```

Comment:
Also this should stop rounds from failing to start because there're fewer than 25 people, which is what the LPO wants. LPO isn't _needed_ for a round to work, so it's okay if we don't cancel the whole round because we don't have enough people.

Reply:
```suggestion
    cancelPresetOnTooFewPlayers: false
```